modules/ROOT/pages/sap/sap-connector-reference-51.adoc

```diff
@@ -0,0 +1,657 @@
+| Queue Name a| String |  The name of the queue on which the RFC executes. |  | 
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  JOIN_IF_POSSIBLE | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
+|===
+
+
+=== For Configurations
+* <<sap>> 
+
+==== Throws
+* SAP:CONNECTIVITY 
+* SAP:INVALID_CACHE 
+* SAP:INVALID_INPUT 
+* SAP:METADATA_UNAVAILABLE 
+* SAP:NOT_FOUND 
+* SAP:NOT_SUPPORTED 
+* SAP:PARSING 
+* SAP:RETRY_EXHAUSTED 
+* SAP:SYSTEM_BUSY 
+* SAP:TIMEOUT 
+
+[[startsaptransaction]]
+=== Start SAP Transaction
+`<sap:create-transaction-id>`
+
+
+Creates a transaction ID to use as part of future calls
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions. |  JOIN_IF_POSSIBLE | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  #[payload] | 
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  | 
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |String
+|===
+
+=== For Configurations
+* <<sap>> 
+
+==== Throws
+* SAP:CONNECTIVITY 
+* SAP:INVALID_CACHE 
+* SAP:INVALID_INPUT 
+* SAP:METADATA_UNAVAILABLE 
+* SAP:NOT_FOUND 
+* SAP:NOT_SUPPORTED 
+* SAP:PARSING 
+* SAP:RETRY_EXHAUSTED 
+* SAP:SYSTEM_BUSY 
+* SAP:TIMEOUT 
+
+
+[[syncRfc]]
+=== Synchronous Remote Function Call
+`<sap:sync-rfc>`
+
+
+Executes a BAPIFunction over a synchronous remote function call (sRFC). A synchronous RFC requires both the systems (client and server) to be available at the time of communication or data transfer. sRFC is the most common type and is used when a result is required immediately after the execution of sRFC.  sRFC is a means of communication between systems where acknowledgments are required. 
+
+The resources of the source system wait at the target system and ensure that they deliver the message or data with ACKD. The Data is consistent and reliable for communication.  If the target system is not available, the source system resources wait until the target system is available. This can lead to the processes of the source system going into Sleep/RFC/CPIC mode at the target systems, which blocks these resources.  sRFC is best used for communication between systems, and communication between a SAP web application server and a SAP GUI.
+
+
+==== Parameters
```

Comment:
Starting here and to the end, there are a lot of missing periods in the descriptions.